src/main/java/uk/gov/moj/sdt/request/CMCFeedback.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.moj.sdt.request;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class CMCFeedback {
+
+    private int errorCode;
+    private String claimNumber;
+    private String errorText;
+    @JsonFormat(pattern="yyyy-MM-dd")
+    protected Date issueDate;
+    @JsonFormat(pattern="yyyy-MM-dd")
+    protected Date serviceDate;
+    @JsonFormat(pattern="yyyy-MM-dd")
+    protected Date judgmentEnteredDate;
+    @JsonFormat(pattern="yyyy-MM-dd")
+    protected Date firstPaymentDate;
+    private String warrantNumber;
+    private String enforcingCourtCode;
+    private String enforcingCourtName;
+    private Long fee;
+    private String judgmentWarrantStatus;
+    private String processingStatus;
```

Comment:
Cosmetic: Could you add blank lines between the field declarations, just to make it a bit easier to read?  At the very least, add them before and after the fields annotated with JsonFormat so it makes it clear that the annotation is for those fields, i.e.

```java
private String errorText;

@JsonFormat(pattern="yyyy-MM-dd")
protected Date issueDate;

@JsonFormat(pattern="yyyy-MM-dd")
protected Date serviceDate;

@JsonFormat(pattern="yyyy-MM-dd")
protected Date judgmentEnteredDate;

@JsonFormat(pattern="yyyy-MM-dd")
protected Date firstPaymentDate;

private String warrantNumber;
```

Reply:
added blank liines